src/Features/Core/Portable/ExtractInterface/ExtractInterfaceCodeAction.cs

```diff
@@ -16,8 +16,6 @@ internal class ExtractInterfaceCodeAction : CodeActionWithOptions
     {
         private readonly ExtractInterfaceTypeAnalysisResult _typeAnalysisResult;
         private readonly AbstractExtractInterfaceService _extractInterfaceService;
-        private readonly Task<IEnumerable<CodeActionOperation>> _taskReturningNoCodeActionOperations = SpecializedTasks.EmptyEnumerable<CodeActionOperation>();
-
```

Comment:
There is no code fix for IDE0052 (remove unread private members), this was manually updated. For large number of cases, this would require user to manually update the statements that write to the members, so an automated code fix is not possible. I can restore the blank line if you feel it is desirable.